draft/XXXX-table-metadata/README.md

```diff
@@ -0,0 +1,80 @@
+---
+RFC: XXXX
+Author: Jacob Nesbitt
+Status: draft
+Created: 2020-10-29
+Last-Modified: 2020-10-29
+---
+
+# RFC XXXX: Table/Graph Metadata
+
+Currently, Multinet doesn't store any metadata about tables (or graphs), chiefly type information. This RFC outlines a way to specify, store and retrieve not just type information, but generalized metadata about a table/graph.
+
+## Motivation
+
+The main motivation for this is storing type information for each column in a table. We now have at least two use cases for storing this information.
+
+1. Basic computation/comparison. If we attempt to use these tables for creating graphs and deriving useful data, it's necessary that we store type information, so that we are able to properly compare and compute certain values. For example, if you have a column of numbers, but we store them as strings (the current behavior), then if you try to do comparisons of these column values between different entries, you either will be doing incorrect string comparisons, or will simply not be able to do the comparisons/computations you need to.
+
+2. Providing this information to other applications, so that they do not have to do this data inspection themselves, or if this information is required by those applications. Specifically, [UpSet](https://github.com/VCG/upset) is the first application for which this will be used.
```

Comment:
```suggestion
2. Providing this information to other applications, so that they do not have to do this data inspection themselves, or if this information is required by those applications. Specifically, [UpSet2](https://github.com/visdesignlab/upset2) is the first application for which this will be used.
```